tests/benchmarks/test_fs.py

```diff
@@ -3,13 +3,20 @@
 import shutil
 from reflink import reflink as pyreflink
 from reflink.error import ReflinkImpossibleError
-from dvc_objects.fs.system import reflink, hardlink, symlink
+from dvc_objects.fs.system import reflink, hardlink, symlink, _reflink
 
 NLINKS = 10000
 
 
 @pytest.mark.parametrize(
-    "link", [pytest.param(pyreflink, id="pyreflink"), reflink, hardlink, symlink]
+    "link",
+    [
+        pytest.param(pyreflink, id="pyreflink"),
+        reflink,
+        hardlink,
+        symlink,
+        _reflink,
```

Comment:
I am not sure if we need to benchmark this, but I added this anyway since it makes benchmarks a bit fair.